locales/pt_BR/LC_MESSAGES/duckduckgo.po

```diff
@@ -2334,7 +2326,7 @@ msgstr "Mais Informações"
 
 msgctxt "Ads GDPR, internal use only. Do not change"
 msgid "More Info"
-msgstr ""
+msgstr "Mais informações"
```

Comment:
@marcellohalfeld Unfortunately at the moment we don't accept translations for this string. Strings related to legal or contractual obligations must follow a different route so that our legal team can vet them (that's why we added "Do not change" in the context). I know this is just "More Info", but I cannot approve the PR. Please remove this string. 🙂